SPECS/openssl/openssl.spec

```diff
@@ -20,6 +20,8 @@ Source7: renew-dummy-cert
 Source9: configuration-switch.h
 Source10: configuration-prefix.h
 Source14: 0025-for-tests.patch
+# CVE only applies when Apache HTTP Server version 2.4.37 or less: https://nvd.nist.gov/vuln/detail/CVE-2019-0190
+Patch1:   CVE-2019-0190.nopatch
```

Comment:
I think we have moved to not bother with `Patch#:` the nopatch files. In fact I think we can also just dispute via astrolabe.